src/main/java/nl/thehyve/ocdu/models/OCEntities/PersonIDUsage.java

```diff
@@ -0,0 +1,10 @@
+package nl.thehyve.ocdu.models.OCEntities;
+
+/**
+ * Created by Jacob Rousseau on 19-Jul-2016.
+ * Copyright CTMM-TraIT / NKI (c) 2016
+ */
+public enum PersonIDUsage {
```

Comment:
There is no need for this class, there is already:  ProtocolFieldRequirementSetting
inside OcDefinitions package